prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -0,0 +1,92 @@
+package gov.cdc.prime.router.tokens
+
+import com.google.common.net.HttpHeaders
+import com.microsoft.azure.functions.ExecutionContext
+import com.microsoft.azure.functions.HttpRequestMessage
+import com.microsoft.azure.functions.HttpResponseMessage
+import gov.cdc.prime.router.Organization
+import gov.cdc.prime.router.azure.*
+import org.apache.logging.log4j.kotlin.Logging
+
+class OktaAuthentication(private val minimumLevel: PrincipalLevel = PrincipalLevel.USER): Logging {
+    private val missingAuthorizationHeader = HttpUtilities.errorJson("Missing Authorization Header")
+    private val invalidClaim = HttpUtilities.errorJson("Invalid Authorization Header")
+
+    fun getAccessToken(request: HttpRequestMessage<String?>): String? {
```

Comment:
Consider putting this function into HttpUtilitlies so other people can use it. @jimduff-usds for example.

Reply:
agree on this, good idea to have it there, it will be useful

Reply:
agree on this, good idea to have it there, it will be useful